src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx

```diff
@@ -88,6 +88,28 @@ export default class AlterationEnrichmentContainer extends React.Component<
         return this.props.groups[1];
     }
 
+    @computed get group1QueriedCasesCount() {
+        let caseIds: Set<string> = new Set(
+            this.group1.samples.map(sample =>
+                this.props.patientLevelEnrichments
+                    ? sample.uniquePatientKey
+                    : sample.uniqueSampleKey
+            )
+        );
+        return caseIds.size;
+    }
+
+    @computed get group2QueriedCasesCount() {
```

Comment:
it seems confusing that "cases" will sometimes mean samples, right? i mean, if we're looking at sampleLevelENrichment, this count will a count of samples.  i think "case" typically means patient in our system.

Reply:
i know we're racing to get this done and we already use "case" generically below.  just want to make sure i understand.